unity/Assets/Scripts/StochasticRemoteFPSAgentController.cs

```diff
@@ -238,44 +248,68 @@ public class StochasticRemoteFPSAgentController : BaseFPSAgentController {
             actionFinished(success: true);
         }
 
-        public override void MoveAhead(ServerAction action) {
-            action.x = 0.0f;
-            action.y = 0;
-            action.z = 1.0f;
-            MoveRelative(action);
+        public void MoveAhead(
```

Comment:
I was under the impression that we made these changes somewhere at some point. @mattdeitke Did you ever incorporate similar changes somewhere? Do I recall correctly? If yes we should make sure we are consistent with those changes.

Reply:
I didn't add `Move{Ahead/Left/Right/Back}` to the arm agent. I basically left a big TODO, which Luca is doing in this PR :)